src/services/recim/team.ts

```diff
@@ -69,11 +69,14 @@ type PatchTeam = {
 };
 
 //Team Routes
+const getTeams = (active: boolean): Promise<Team[]> => {
+  if (active) return http.get(`recim/Teams?active=${active}`);
+  return http.get(`recim/Teams`);
+};
```

Comment:
There is a bug in the API, I assume that this route should automatically give all teams if there is no active tag, but right now it simply has active = true/false

Reply:
fixed here: 
https://github.com/gordon-cs/gordon-360-api/pull/793